networking_aci/plugins/ml2/drivers/mech_aci/driver.py

```diff
@@ -50,7 +51,7 @@ def initialize(self):
     def _setup_rpc(self):
         """Initialize components to support agent communication."""
         self.endpoints = [
-            rpc_api.AgentRpcCallback(),
+            rpc_api.AgentRpcCallback(self.db),
```

Comment:
Is there something missing to support passing the DBPlugin to `AgentRpcCallback`? I also found another instances of `common.get_address_scope_name()` [here](https://github.com/sapcc/networking-aci/blob/77f6c64b6bb05ef7e11b3bb4bdff02947212057d/networking_aci/plugins/ml2/drivers/mech_aci/rpc_api.py#L150), that's not switched over:

Reply:
Ehrm, yes, you're right, I missed something with my `git add -p`